bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -174,6 +205,13 @@ public CheckAndMutateRowResponse checkAndMutateRow(CheckAndMutateRowRequest requ
       IS_RETRYABLE_MUTATION, request);
   }
 
+  private ClientCall<MutateRowsRequest, MutateRowsResponse>
+      createMutateRowsCall(MutateRowsRequest request) {
+    expandPoolIfNecessary(this.bigtableOptions.getChannelCount());
```

Comment:
This gets called by blocking and async versions of mutateRows, can we move expandPoolIfNecessary to the mutateRowsAsync()?